module/actor.js

```diff
@@ -606,6 +606,25 @@ export class cyberpunkredActor extends Actor {
     }
   }
 
+  isCritInjury(roll) {
```

Comment:
I just realized I defined these in a class called actor, and actor.isCritInjury might be pretty confusing. Should I rename them to something like checkCritInjury instead, or move them to a different class?

Reply:
I think "check" would make more sense than "is"... just to make it easier when module devs go through trying to figure it out.

Reply:
Also, please be aware I just pushed some small changes to actor.js. None in the roll section, all in the data prep section